### DISCUSSION TITLE
Increasing visibility of selected options

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [dmitry-s93/MControlCenter#105] [Ideas]
In the current scenario, to know which option is currently selected, you need to open the app. But, if it's possible to add the ability to show a tick in the current selection, like the cooler boost mode, then it can save some extra mouse clicks.

**Comment by dmitry-s93** [comment]
There are plans to display the current options in a popup when hovering over the tray icon

**Comment by mutchiko** [comment]
Implemented in https://github.com/dmitry-s93/MControlCenter/pull/287

<img width="658" height="440" alt="Screenshot_20260321_181340" src="https://github.com/user-attachments/assets/db9b10af-1b53-48d3-95ba-4bab6535fd6a" />
